README.md

```diff
@@ -16,6 +16,14 @@ Scrumple is based on a now-deleted tool called Pax, but we have added support fo
 
 This is a developer tool, not production software. It's designed to give the developer super-fast feedback when building components, not for building a production application.
 
+## Installation
```

Comment:
```suggestion
## Installation

Scrumple requires `npm` to be installed.
```

Reply:
I think Scrumple requires npm, based on this line -- https://github.com/Financial-Times/scrumple/pull/54/files#file-src-main-rs-L71

Reply:
it doesn't. that's only for tests and benchmarks. and it isn't even actually required in that test.